doc/classes/CanvasItem.xml

```diff
@@ -415,7 +415,7 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the node is in the [SceneTree] and is visible on-screen.
+				Returns [code]true[/code] if the node is present in the [SceneTree], its [code]visible[/code] property is [code]true[/code] and its inherited visibility is also [code]true[/code].
```

Comment:
Maybe replace `[code]visible[/code]` with `[member visible]` so it generates a link to the property.

Reply:
Good idea, done!